rtcpeerconnection.js

```diff
@@ -617,11 +617,15 @@ module.exports = function(window, edgeVersion) {
     var params = getCommonCapabilities(transceiver.localCapabilities,
         transceiver.remoteCapabilities);
     if (send && transceiver.rtpSender) {
-      params.encodings = transceiver.sendEncodingParameters;
```

Comment:
i think keeping this but doing `params.encodings = transceiver.sendEncodingParameters || []` is better. However I don't understand how sendEncodingParameters could not be set?

Reply:
OK, I think I might have just done the (possibly un-needed) `sendEncodingParameters` change to stay in step with the (neecessary) `recvEncodingParameters` options below. I believe I also tried `transceiver.recvEncodingParameters || []` and found that Edge fails on an empty array.

The `sendEncodingParamaters` being browser originated MUST be suitable for the browsers needs by definition - the `recvEncodingParameters` comes from an unknown other party, which is what is the issue here.